Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:lts-alpine as vue-build
+WORKDIR /app
+COPY ./client/package.json ./
+RUN npm install
+COPY ./client/ .
+RUN npm run build
+
+FROM node:lts-alpine AS server-build
+WORKDIR /app
+COPY ./server/package.json ./
+RUN npm install
+COPY ./server .
+COPY --from=vue-build /app/dist ./dist
```

Comment:
theoretically we can change this to:
`COPY --from=vue-build /app/dist ./src/public`

then we don't need to change one line in bottom

Reply:
I would skip it for now because it makes no big difference